scripts/globals/pasteventwatcher.lua

```diff
@@ -0,0 +1,220 @@
+-----------------------------------
+-- Past Event Watcher NPCs
+-----------------------------------
+
+-----------------------------------
+
+require("scripts/globals/quests")
+require("scripts/globals/missions")
+require("scripts/globals/zone")
+
+pasteventwatcher = {}
+
+local gobCS = -- add the goblin cs to this table
+{
+    [102] = 126, -- La Theine Plateau
+    [106] = 247, -- North Gustaberg
+    [107] = 905, -- South Gustaberg
+    [119] = 35,  -- Meriphataud Mountains
+    [122] = 6,   -- Ro'Maeve
+    [162] = 10,  -- Castle Zvahl Keep
+    [167] = 100, -- Bostaunieux Oubliette
+    [198] = 67,  -- Maze of Shakhrami
+    [200] = 61,  -- Garlaige Citadel
+}
+
+function pasteventwatcher.rewatch(player, trigger)
+    local zone = player:getZoneID()
+    local ID = zones[zone]
+    local options = 1
+    local cs = -- add checks to this table
+    {
+        [102] = -- La Theine Plateau
```

Comment:
Death to magic numbers in this whole table.

```lua
local cutscenesList = 
{
    [dsp.zone.LA_THEINE_PLATEAU] = {},
    [dsp.zone.NORTH_GUSTABERG] = {},
    etc
}
```

Reply:
fixed 😸